musicrando.py

```diff
@@ -84,7 +84,10 @@ def music_rando(self):
             ):
                 track_len = self.musiclist[original_track]["audiolen"]
             else:
-                track_len = self.musiclist[new_track]["audiolen"]
+                if self.placement_file.options["allow-custom-music"]:
+                    track_len = 2147483647  # 0x7F FF FF FF - 2GB
```

Comment:
```suggestion
                    track_len = 0x7FFFFFFF # - 2GB
```
Makes more sense as hex